docs/reference/contributing/connectivity/CellularInterface.md

```diff
@@ -164,50 +85,72 @@ typedef enum {
 #define MDM_PIN_POLARITY            ACTIVE_HIGH_POLARITY
 
 ```
-The current implementation does not use all pins, but you must define all of them.
-
-3. **Implement `onboard_modem_api.h`** Provide an implementation of `onboard_modem_api.h`. We provide an [example implementation](https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_NXP/TARGET_LPC176X/TARGET_UBLOX_C027/onboard_modem_api.c).
-
-###### Providing module modem API
-
-Only valid when **Case 4** is applicable.
 
-- If the modem is already ready to use via the UART, it may be possible to use `UARTCellularInterface` directly. Just pass its constructor the necessary pin information for the module connected to your board.
+You must define all pins. Implement `onboard_modem_api.h`. The target board must provide an implementation of the [onboard_modem_API](https://github.com/ARMmbed/mbed-os/blob/master/features/netsocket/cellular/onboard_modem_api.h). Please see the [example implementation ](https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_NXP/TARGET_LPC176X/TARGET_UBLOX_C027/onboard_modem_api.c).
```

Comment:
We don't like to link to GitHub, so we need to find all of the GitHub links in the Doxygen once it's generated.